composer.json

```diff
@@ -128,9 +128,9 @@
         "symfony-cmf/routing": "^2.0.3",
         "symfony-cmf/routing-bundle": "^2.0.3",
         "symfony/browser-kit": "^4.4.0",
-        "symfony/debug": "^4.4.0",
```

Comment:
could you please perform these changes also in `composer.json` files in `framework` package and in `project-base`, please? :slightly_smiling_face: